app/views/internal/path_redirects/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= link_to "Back to All Path Redirects", internal_path_redirects_path, class: "btn btn-primary float-right" %>
+
+<h2>Path Redirect - ID:  <%= @path_redirect.id %></h2>
+<% badge_type =
+     if @path_redirect.version <= 3
+       "success"
+     elsif @path_redirect.version <= 5
+       "warning"
+     else
+       "danger"
+     end %>
+<h4 class="d-inline-block mb-5">
+  Version:
+  <span class="align-top badge badge-<%= badge_type %>"><%= @path_redirect.version %></span>
+</h4>
```

Comment:
May want to include a note (warning?) here about not updating paths once you hit a certain version?

Reply:
Love that - updated!

<img width="1267" alt="Screen Shot 2020-06-01 at 6 29 28 PM" src="https://user-images.githubusercontent.com/15987080/83461227-2e49d000-a436-11ea-99e6-121fb4bad126.png">